backend/audit/templates/audit/submission-progress.html

```diff
@@ -207,7 +222,10 @@ <h1 class="border-base-light margin-top-0">{{ auditee_name | default_if_none:''
 				<div class="usa-icon-list__content {% if secondary_auditors_workbook.created is True %} text-green {% else %} text-base-dark {% endif %}">
 					<a class="usa-link text-bold"
 						href="/report_submission/secondary-auditors/{{ report_id | default_if_none:'' }}">Upload the Secondary Auditors workbook</a>
-				</div>
+						<p class="margin-0">
+							This workbook is only necessary if multiple auditors did the audit work. In these cases, the secondary auditors should enter their  information in this workbook.
```

Comment:
`their  information`
Double space! They display as one space anyway, so it's not impactful.

Reply:
Fixed, thanks!